keps/sig-auth/3257-trust-anchor-sets/README.md

```diff
@@ -0,0 +1,1184 @@
+in back-to-back releases.
+
+**For non-optional features moving to GA, the graduation criteria must include
+[conformance tests].**
+
+[conformance tests]: https://git.k8s.io/community/contributors/devel/sig-architecture/conformance-tests.md
+
+#### Deprecation
+
+- Announce deprecation and support policy of the existing flag
+- Two versions passed since introducing the functionality that deprecates the flag (to address version skew)
+- Address feedback on usage/changed behavior, provided on GitHub issues
+- Deprecate the flag
+-->
+
+### Upgrade / Downgrade Strategy
+
+At present, there are no upgrade / downgrade concerns.  The ClusterTrustBundle feature gate controls overall availability of the feature.
+
+### Version Skew Strategy
+
+Both kubelet and kube-apiserver will need to be at 1.26 for the full featureset to be present.  If only kube-apiserver is at 1.26 and kubelet is lower, then the the pod mounting feature will be cleanly unavailable, but all other aspects of the feature will work.
+
+## Production Readiness Review Questionnaire
+
+<!--
+
+Production readiness reviews are intended to ensure that features merging into
+Kubernetes are observable, scalable and supportable; can be safely operated in
+production environments, and can be disabled or rolled back in the event they
+cause increased failures in production. See more in the PRR KEP at
+https://git.k8s.io/enhancements/keps/sig-architecture/1194-prod-readiness.
+
+The production readiness review questionnaire must be completed and approved
+for the KEP to move to `implementable` status and be included in the release.
+
+In some cases, the questions below should also have answers in `kep.yaml`. This
+is to enable automation to verify the presence of the review, and to reduce review
+burden and latency.
+
+The KEP must have a approver from the
+[`prod-readiness-approvers`](http://git.k8s.io/enhancements/OWNERS_ALIASES)
+team. Please reach out on the
+[#prod-readiness](https://kubernetes.slack.com/archives/CPNHUMN74) channel if
+you need any help or guidance.
+-->
+
+### Feature Enablement and Rollback
+
+###### How can this feature be enabled / disabled in a live cluster?
+
+This feature is gated by the ClusterTrustBundle feature gate.  The feature gate controls:
+
+* Whether or not kube-apiserver will accept operations on ClusterTrustBundle
+  objects.
+
+* Whether or not kube-apiserver will accept webhook and aggregated API server
+  backend configs that refer to ClusterTrustBundle objects.
+
+* Whether or not kubelet will accept pemTrustAnchors projected volume sources in
+  pod specs.
+
+###### Does enabling the feature change any default behavior?
+
+Enabling the feature does not change any default behavior.
+
+###### Can the feature be disabled once it has been enabled (i.e. can we roll back the enablement)?
+
+Yes, if the ClusterTrustBundle feature gate is disabled, the cluster will largely continue to function properly.
```

Comment:
need to indicate how the trust by server components changes when their hooks go from "missing cabundle and present clustertrustbundle" to simply "missing ca bundle"

Reply:
I think this is now handled by specifying that both fields may be set but that clustertrustbundlename takes precedence.